mlflow/projects/__init__.py

```diff
@@ -145,7 +147,34 @@ def _run(uri, experiment_id, entry_point="main", version=None, parameters=None,
             work_dir=work_dir, entry_point=entry_point, parameters=parameters,
             experiment_id=experiment_id,
             use_conda=use_conda, storage_dir=storage_dir, run_id=active_run.info.run_id)
-    supported_backends = ["local", "databricks"]
+    elif backend == "kubernetes":
+        from mlflow.projects import kubernetes as kb
+        tracking.MlflowClient().set_tag(active_run.info.run_uuid,
```

Comment:
I think the project env should be `docker` right? Maybe we need a new tag called project execution mode.

Reply:
Yes you're right. I've created a tag called `backendMode`, so we can have it with the same name as parameter okay?